api/v1/mongodbcommunity_types.go

```diff
@@ -355,6 +356,12 @@ type AgentConfiguration struct {
 	LogLevel LogLevel `json:"logLevel"`
 	// +optional
 	MaxLogFileDurationHours int `json:"maxLogFileDurationHours"`
+	// +optional
+	// LogRotate if enabled, will enable LogRotate for all processes.
+	LogRotate *automationconfig.LogRotate `json:"logRotate,omitempty"`
```

Comment:
Since LogRotate requires SystemLog, would it be better to add this as a validation step in the webhook?

Reply:
yes! I can add it in community later!